src/vs/workbench/contrib/terminal/browser/xterm/xtermTerminal.ts

```diff
@@ -567,13 +567,15 @@ export class XtermTerminal extends DisposableStore implements IXtermTerminal {
 		const cursorColor = theme.getColor(TERMINAL_CURSOR_FOREGROUND_COLOR) || foregroundColor;
 		const cursorAccentColor = theme.getColor(TERMINAL_CURSOR_BACKGROUND_COLOR) || backgroundColor;
 		const selectionColor = theme.getColor(TERMINAL_SELECTION_BACKGROUND_COLOR);
```

Comment:
nit: unrelated to this PR but perhaps this should be more explicit now that you have the foreground token.

E.g. `selectionBackgroundColor`

Reply:
https://github.com/microsoft/vscode/pull/149839